modules/meeting/app/components/meetings/side_panel/details_dialog_component.html.erb

```diff
@@ -0,0 +1,10 @@
+<%=
+  render(Primer::Alpha::Dialog.new(
```

Comment:
There are some issues with the async dialog:

* When I submit a form with an error (e.g. with empty duratio), there is no error message shown. The dialog can't even be closed via the buttons any more.
* When I open the dialog, and close it via the buttons, everything is fine. But if I open the dialog again, I cannot close it any more.